service/matching/physical_task_queue_key.go

```diff
@@ -51,10 +53,16 @@ type (
 	// Physical task queues with a version set or build ID are called "versioned". The ones without a version set
 	// or build ID are called "unversioned". A physical queue cannot have both version set and build ID.
 	PhysicalTaskQueueKey struct {
-		partition  tqid.Partition
+		partition tqid.Partition
+		version   PhysicalTaskQueueVersion
+	}
+
+	PhysicalTaskQueueVersion struct {
```

Comment:
wasn't this in the previous pr? do you need to merge so we only see the diff from this one?

Reply:
yes it was, and the previous PR is merged, it might be that Github has not updated the diff?! I'm going to merge some changes now and hopefully the diff gets refreshed.